shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -385,7 +390,17 @@ private void copyPath(AlluxioURI srcPath, AlluxioURI dstPath) throws AlluxioExce
 
       FileOutStream os = null;
       try (Closer closer = Closer.create()) {
-        os = closer.register(mFileSystem.createFile(dstPath));
+        FileWriteLocationPolicy locationPolicy;
+        try {
+          locationPolicy =
+              CommonUtils.createNewClassInstance(Configuration.<FileWriteLocationPolicy>getClass(
+                  PropertyKey.USER_FILE_COPY_FROM_LOCAL_WRITE_LOCATION_POLICY), new Class[] {},
+                  new Object[] {});
+        } catch (Exception e) {
+          throw Throwables.propagate(e);
```

Comment:
Just for your info, `Throwables.propagate(e)` has been depreciated by Google. Can you change this to be
```java
throw new RuntimeException(e);
```